projects/plugins/protect/.gitattributes

```diff
@@ -13,6 +13,7 @@ vendor/autoload_packages.php                    production-include
 vendor/automattic/**                            production-include
 vendor/composer/**                              production-include
 vendor/jetpack-autoloader/**                    production-include
+vendor/wikimedia/aho-corasick/**                production-include
```

Comment:
Why is this being added?

Reply:
The discussion can be found here: p1670548960264089-slack-CBG1CP4EN

TL;DR as I understood it, that package dependency wasn't being added to Protect's list of dependencies and it was causing a fatal somewhere down line.